.circleci/config.yml

```diff
@@ -155,10 +155,7 @@ jobs:
           keys:
             - ccache-release-macos-{{ .Branch }}-{{ checksum "conanfile.txt" }}
             - ccache-release-macos-{{ checksum "conanfile.txt" }}
-      - run: |
-          mkdir -p build
-          cd build
-          cmake ..
+      - run: cmake -B build -DENABLE_SINGLE_FILES_WERROR=OFF
```

Comment:
with the new clang there's tons of more warnings treated as errors

Reply:
also no way to enable this again anytime soon, it's also robin hood which causes the errors/warnings